sampleData/sampleData.js

```diff
@@ -0,0 +1,1610 @@
+export default sampleData = {
```

Comment:
this has the wrong import statement, needs to be module.exports = {
sampleData: .....
}

think it snuck in because the file was large and didn't load